test/tests/zoteroPaneTest.js

```diff
@@ -310,4 +310,50 @@ describe("ZoteroPane", function() {
 			assert.lengthOf(Object.keys(conditions), 2);
 		});
 	});
+	
+	describe("#onCollectionSelected()", function() {
+		var cv;
+		
+		before(function* () {
+			cv = zp.collectionsView;
```

Comment:
The `loadZoteroPane` in the last test will break `cv` for any tests that get added after, so this should be moved to `beforeEach`.